### DISCUSSION TITLE
Message d'erreur dans Orca

### DISCUSSION THREAD

**Discussion by DJe73260** [Guilouz/Creality-Helper-Script-Wiki#590] [Issues]
Salut
J'ai ce message de la part de Orca.
Je te rassure, cela ne gène pas le fonctionnement
J'ai fait les mises à jour et je ne vois plus quoi essayer

LE MESSAGE 
"Moonraker avertissements .
Server configuration error: Configuration File Not Found: '/usr/data/printer_data/config/moonraker.conf''
Loaded server from most recent working configuration: '/usr/data/printer_data/config/.moonraker.conf.bkp'
Please fix the issue in moonraker.conf and restart the server." 

As tu une solution ?

**Comment by Guilouz** [comment]
Ça signifie que tu n’as plus de fichier moonraker.conf ou qu’il n’est pas valide donc il a chargé le backup.

**Comment by DJe73260** [comment]
Se sont des fichiers cachés ? Quand le liste les fichier (ls) je n'ai rien Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : Cyril Guislain ***@***.***> Date : 26/05/2024  21:17  (GMT+01:00) À : Guilouz/Creality-Helper-Script-Wiki ***@***.***> Cc : DJe73260 ***@***.***>, Author ***@***.***> Objet : Re: [Guilouz/Creality-Helper-Script-Wiki] Message d'erreur dans Orca (Discussion #590) 
Ça signifie que tu n’as plus de fichier moonraker.conf ou qu’il n’est pas valide donc il a chargé le backup.

**Reply by Guilouz** [reply]
Re télécharge ce fichier : https://github.com/Guilouz/Creality-Helper-Script/blob/main/files/moonraker/moonraker.conf

Et décommente les partie dont tu as besoin

**Comment by DJe73260** [comment]
OkMerci et bravoEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : Cyril Guislain ***@***.***> Date : 29/05/2024  17:13  (GMT+01:00) À : Guilouz/Creality-Helper-Script-Wiki ***@***.***> Cc : DJe73260 ***@***.***>, Author ***@***.***> Objet : Re: [Guilouz/Creality-Helper-Script-Wiki] Message d'erreur dans Orca (Discussion #590) 
Re télécharge ce fichier : https://github.com/Guilouz/Creality-Helper-Script/blob/main/files/moonraker/moonraker.conf
Et décommente les partie dont tu as besoin
